File: SoalA/C++/CPU.cpp
#include <iostream>
#include <string>

using namespace std;

class CPU
{
    /* membuat atribut */
private:
    string nameCPU;
    int priceCPU;

public:
    /* consturctor */
    CPU(int priceCPU, string nameCPU)
    {
        this->priceCPU = priceCPU;
        this->nameCPU = nameCPU;
    }

    CPU()
    {
    }

    /* Membuat get set method harga CPU */
    void setPriceCPU(int priceCPU)
    {
        this->priceCPU = priceCPU;
    }
    int getPriceCPU()
    {
        return this->priceCPU;
    }

    /* Membuat get set method nama CPU */
    void setNameCPU(string nameCPU)
    {
        this->nameCPU = nameCPU;
    }
    string getNameCPU()
    {
        return this->nameCPU;
    }

    /* menampilkan data CPU */
    void printCPU()
    {
        cout << "====================================================" << endl;
        cout << "CPU Price(IDR)     : " << this->getPriceCPU() << endl;
        cout << "CPU Name           : " << this->getNameCPU() << endl;
        cout << "====================================================" << endl;
    }

    /* destructor */
    ~CPU()
    {
    }
};

File: SoalB/Pyhton/CPU.py
class CPU():
    # atribut
    __priceCPU = 0
    __nameCPU = ""

    # constructor
    def __init__(self):
        self.__priceCPU = 0
        self.__nameCPU = ""

    # membuat get set method harga cpu
    def setPriceCPU(self, priceCPU):
        self.__priceCPU = priceCPU

    def getPriceCPU(self):
        return self.__priceCPU

    # membuat get set method nama cpu
    def setNameCPU(self, nameCPU):
        self.__nameCPU = nameCPU

    def getNameCPU(self):
        return self.__nameCPU

    # menampilkan data cpu
    def printCPU(self):
        print("---------------------------------------")
        print("CPU Price(IDR)       : " + str(self.getPriceCPU()))
        print("CPU Name             : " + str(self.getNameCPU()))
        print("---------------------------------------")
